src/platform/forms/tests/save-in-progress/01-sip-autosave.cypress.spec.js

```diff
@@ -0,0 +1,125 @@
+      enrollmentDate: '2018-01-24T00:00:00.000-06:00',
+      preferredFacility: '987 - CHEY6',
+      parsedStatus: 'none_of_the_above',
+    });
+    cy.title().should('contain', 'Apply for Health Care | Veterans Affairs');
+    cy.get('.main .usa-button-primary', { timeout: Timeouts.slow }).should(
+      'be.visible',
+    );
+    cy.injectAxeThenAxeCheck();
+    cy.get('.main .usa-button-primary')
+      .first()
+      .click();
+
+    cy.url().should('not.contain', '/introduction');
+    cy.url().should('contain', '/veteran-information/birth-information');
+
+    cy.get('.schemaform-sip-save-link', { timeout: Timeouts.normal }).should(
+      'be.visible',
+    );
+    cy.get('#root_veteranSocialSecurityNumber').should(
+      'have.attr',
+      'value',
+      '123445544',
+    );
+    cy.fill(
+      'input[name="root_view:placeOfBirth_cityOfBirth"]',
+      'Northhampton, MA',
+    );
+    cy.get('.saved-success-container', { timeout: Timeouts.normal }).should(
+      'be.visible',
+    );
+    cy.get('.main .usa-button-primary').click();
+    cy.get('.schemaform-sip-save-link', { timeout: Timeouts.normal }).should(
+      'be.visible',
+    );
+    cy.intercept('PUT', '/v0/in_progress_forms/1010ez', {
+      statusCode: 500,
+      body: {},
+    });
+
+    cy.fill('input[name="root_view:placeOfBirth_cityOfBirth"]', 'Amherst, MA');
+    cy.get('.usa-alert-error', { timeout: Timeouts.normal }).should(
+      'be.visible',
+    );
+
+    cy.url().should('contain', 'birth-information');
+    cy.get('.usa-alert-error').should(
+      'contain',
+      'We’re sorry, but we’re having some issues and are working to fix them',
+    );
+
+    // Recover and save after errors
+    /* eslint-disable camelcase */
+
+    cy.intercept('PUT', '/v0/in_progress_forms/1010ez', {
+      data: {
+        attributes: {
+          metadata: {
+            version: 0,
+            returnUrl: '/veteran-information/birth-information',
+            savedAt: 1498588443698,
+            expires_at: moment()
+              .add(1, 'day')
+              .unix(),
+            last_updated: 1498588443,
+          },
+        },
+      },
+    });
+    /* eslint-enable camelcase */
+
+    cy.fill('input[name="root_view:placeOfBirth_cityOfBirth"]', 'Florence, MA');
+    cy.get('.saved-success-container', { timout: Timeouts.normal }).should(
+      'be.visible',
+    );
+
+    cy.url().should('contain', '/veteran-information/birth-information');
+
+    // fail to save a form because signed out
+    // Can't recover from this because it logs you out and we'd have to log in again
+
+    cy.get('.main .usa-button-primary').click();
+    cy.get('.schemaform-sip-save-link', { timeout: Timeouts.normal });
+    cy.intercept('PUT', '/v0/in_progress_forms/1010ez', {
+      body: {},
+      statusCode: 401,
+    });
+    cy.fill('input[name="root_view:placeOfBirth_cityOfBirth"]', 'Amherst, MA');
+    cy.get('.usa-alert-error', { timeout: Timeouts.normal });
```

Comment:
Same thing as a previous comment, but I believe `cy.get` defaults to `Timeouts.normal` 🙂

Reply:
I believe our timeouts helper that's being used has `2000` set as the normal presently, where Cypress defaults to a 4000.  In this case, it's shortening that period. I'm not 100% sure it's necessary, as you've stated, however it's another case of just matching what was stable in Nightwatch.

Reply:
Understood! I do think you can omit the `, { timeout: Timeouts.normal }` for each `cy.get` in this case, but I also totally get wanting to achieve parity at first to make sure nothing breaks 👍

Reply:
Giving a shot at removing these and running the full suite to make sure it's still stable.  Will post in the other conversation regarding this when I have an update!  Resolving this one to condense.